hedera-node/src/test/java/com/hedera/services/txns/schedule/ScheduleDeleteTransitionLogicTest.java

```diff
@@ -38,78 +40,96 @@
 
 import java.time.Instant;
 
-import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.FAIL_INVALID;
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_SCHEDULE_ID;
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.OK;
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.SUCCESS;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.collection.IsIterableContainingInOrder.contains;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.mockito.BDDMockito.given;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
 import static org.mockito.Mockito.verify;
 
 @ExtendWith({ LogCaptureExtension.class })
 class ScheduleDeleteTransitionLogicTest {
 	private static final Instant consensusNow = Instant.ofEpochSecond(1_234_567L, 890);
 	private static final ResponseCodeEnum NOT_OK = null;
 	private static final ScheduleID schedule = IdUtils.asSchedule("1.2.3");
+	private static byte[] keyBytes = {0, 1, 0, 1};
+	private JKey adminKey = new JEd25519Key(keyBytes);
+	private Schedule loadedSchedule;
 
-	private ScheduleStore store;
+	private ScheduleModelStore store;
 	private PlatformTxnAccessor accessor;
 	private TransactionContext txnCtx;
 
 	private TransactionBody scheduleDeleteTxn;
 
-	@LoggingTarget
-	private LogCaptor logCaptor;
-
-	@LoggingSubject
 	private ScheduleDeleteTransitionLogic subject;
 
 	@BeforeEach
 	private void setup() {
-		store = mock(ScheduleStore.class);
+		store = mock(ScheduleModelStore.class);
```

Comment:
At some point was mentioned, that we should prefer annotation-based mocking over mocking within the @BeforeEach method. If this is still valid - I think that we should stick to that also for the reason, that unnecessary mocking is not allowed and a correspondent exception is being thrown during test execution.